examples/portfolio/src/pages/projects.astro

```diff
@@ -4,9 +4,13 @@ import Footer from '../components/Footer/index.jsx';
 import Nav from '../components/Nav/index.jsx';
 import PortfolioPreview from '../components/PortfolioPreview/index.jsx';
 
-const projects = Astro.fetchContent('./project/*.md')
+interface Frontmatter {
+  published_at: number;
```

Comment:
I love when TS fixes expose weird naming :)

Can you rename this lowerCamelCase to match `examples/blog`, and also use the same name for both (`publishedAt`? `publishDate`?)